x/src/test.rs

```diff
@@ -17,21 +21,58 @@ pub struct Args {
     #[structopt(long, short)]
     /// Only run unit tests
     unit: bool,
+    #[structopt(long, short, parse(from_os_str))]
+    lcovdir: Option<OsString>,
+    #[structopt(long, short, parse(from_os_str))]
+    htmlcovdir: Option<OsString>,
     #[structopt(name = "TESTNAME", parse(from_os_str))]
     testname: Option<OsString>,
+
     #[structopt(name = "ARGS", parse(from_os_str), last = true)]
     args: Vec<OsString>,
 }
 
 pub fn run(mut args: Args, config: Config) -> Result<()> {
     args.args.extend(args.testname.clone());
 
+    //if we will be be running coverage, set the environment arguments we'll force
+    let runcov = args.lcovdir.is_some() || args.htmlcovdir.is_some();
+    let coverageflags: &[&str] = &[
```

Comment:
If these are env vars then you should be storing them with the type
```suggestion
    let coverageflags: &[(&str, &str)] = &[
        ("key", "value"),
        ("key", "value"),
    ];
```

Reply:
ahhh, the array of tuples will be able to allow the enum to derive Default/Clone.   Nice.

Reply:
stored them as OsStrings, made the transition straight forward.... maybe I don't need to though.  quick check time.

Reply:
done.